### ISSUE TITLE
lsp command

### ISSUE THREAD

**Issue by fladd** [bram85/topydo#256]
In the original todo.sh there is a 'lsp' command to list priorities (and further filter them).

For instance,

```
t lsp B +Work.
```

Would only return tasks with a priority of `(B)` and part of projects that start with `+Work.`

**Comment by bram85** [comment]
topydo already supports filters by priority

topydo ls (B) +Work

(use appropriate escaping for parentheses if applicable).

**Comment by fladd** [comment]
Yes, I am aware of this. It would be a shortcut to not have to escape all the time.

**Comment by rnkn** [comment]
@fladd add this to your config:
~~~
[aliases]
lspa = ls (A)
lspb = ls (B)
...etc.
~~~

**Comment by BioBox** [comment]
If I were to add a `-p` option to the list command that takes a comma separated list of priority expressions, then were would I put the test code for that in `test_list_command.py`? Would such a flag even be acceptable in the first place?

**Comment by davesteele** [comment]
> Would such a flag even be acceptable in the first place?

Sure.

This should probably be the 'ls' command, not 'lsp'.

It would also need to be documented in the tiddlywiki doc.

**Comment by BioBox** [comment]
Okay I wrote the code and some test functions for it here: #305 
This is kind of unnecessary because I didn't know that you could use `(>C)` instead which does exactly what I wanted, but oh well.
